src/osd/PrimaryLogPG.cc

```diff
@@ -3578,9 +3578,10 @@ bool PrimaryLogPG::inc_refcount_by_set(OpContext* ctx, object_manifest_t& set_ch
       }
     }
     if (mop->tids.size()) {
+      mop->cb = new RefCountCallback(ctx, osd_op);
       manifest_ops[ctx->obs->oi.soid] = mop;
       manifest_ops[ctx->obs->oi.soid]->op = ctx->op;
-    }
+    } 
```

Comment:
might want to remove the trailing space.

Reply:
Thanks. Fixed.